libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/accounts/MapboxNavigationAccounts.kt

```diff
@@ -71,6 +73,11 @@ class MapboxNavigationAccounts private constructor() : SkuTokenProvider {
         } ?: resourceUrl
     }
 
+    /**
+     * Returns current SDK SKU token needed for ART.
```

Comment:
Basic question, what's ART? Can we spell it out for the comment?

Reply:
Api Routing Tiles